sdk/extensions/Azure.Extensions.AspNetCore.Configuration.Secrets/src/Azure.Extensions.AspNetCore.Configuration.Secrets.csproj

```diff
@@ -13,7 +13,9 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="Azure.Security.KeyVault.Secrets" />
+    <!-- Use package reference after Package.Data.props has a version higher than 4.7.0 -->
```

Comment:
Can we please open an issue and assign it to the KV team?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/53036